scripts/validation_test.sh

```diff
@@ -18,6 +19,15 @@ trap stop TERM
 
 res=""
 
+function run_new_member() {
+     docker_id=$(docker ps -qf "name=node4")
+    if [ -n "$docker_id" ]; then
+        docker stop $docker_id
```

Comment:
In PR #654, we delete log files when the `stop_all` method is invoked. I think it's not a good practice for debugging. If the validation test failed, it will remove the log file too.